internal/examples/protobuf/examplepb/example.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package uber.yarpc.internal.examples.protobuf.example;
+
+import "go.uber.org/yarpc/yarpcproto/yarpc.proto";
```

Comment:
I want to leave this as is, because you really want a separate package for your generated files anyways.